src/Microsoft.Extensions.Configuration.AzureAppConfiguration/JsonKeyValueAdapter.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Azure.Data.AppConfiguration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.AzureKeyVault;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.FeatureManagement;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Mime;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    internal class JsonKeyValueAdapter : IKeyValueAdapter
+    {
+        private static readonly IEnumerable<string> _excludedJsonContentTypes = new[] {
```

Comment:
>_excludedJsonContentTypes [](start = 52, length = 25)

Since this is `static`, the naming convention would be to use `ExcludedJsonContentTypes` instead of `_excludedJsonContentTypes`.